docker/Dockerfile.template

```diff
@@ -0,0 +1,26 @@
+FROM node:6.10.0
```

Comment:
We could switch to `node:7.9.0-alpine`. It has smaller image footprint. The only issue is more work in installing hugo. I haven't played much with alpine based images.